src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/src/3.cpp

```diff
@@ -0,0 +1,94 @@
+#define SENSOR_DISTANCE 7.0
+#define CONF_SSID "MoDeS3"
+#define PASS "LaborImage"
+
+#include <MQTT_JSON.hpp>
+#include <InfraSensor.hpp>
+#include <SensorStatemachine.hpp>
+#include <RealTimeSelect.hpp>
+#include <RealTimeLength.h>
+#include <realtimespeed.hpp>
+#include <string>
+#include <stdlib.h>
+
+struct Serl {
+  Serl() {
+    Serial.begin(9600);
+  };
+};
+Serl s;

```

Comment:
You expect this object to load first and use it to initialize the Serial port so leave a comment about it.
However if every object would be created _after_ the start of the `main` method, you could just start the Serial at the beginning of the `main`